custom_components/easee/services.yaml

```diff
@@ -1,40 +1,45 @@
 start:
-  description: "Deprecated HA 2023.1.0\nStarts charger. Also authorizes if configured to use authorization."
+  name: "Start"
```

Comment:
My intension was to avoid changes in the parts that are deprecated. OK with me to add friendly names.

Reply:
Yes, just my OCD setting in :)